Makefile

```diff
@@ -19,11 +19,11 @@ BASE_MODULES = \
 # variables
 # ---------
 
-ifeq ($(strip $(DESTDIR)),)
-  INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+INSTALLBASE = $(DESTDIR)/usr/share/gnome-shell/extensions
```

Comment:
Isn't this going to install to `~/usr/share/gnome-shell/extensions` for the non-system/superuser case?

Reply:
Argh I'am an idiot / fixed in 1aea4d3 / Thank you so much for being so attentive.